soc/common/nordic_nrf/Kconfig.peripherals

```diff
@@ -74,6 +74,9 @@ config HAS_HW_NRF_GPIO0
 config HAS_HW_NRF_GPIO1
 	def_bool $(dt_nodelabel_enabled_with_compat,gpio1,$(DT_COMPAT_NORDIC_NRF_GPIO))
 
+config HAS_HW_NRF_GPIO2
```

Comment:
Flip first 2 commits around, should match the order that they are in the upstream PR